src/tests/ftest/scrubber/frequency.py

```diff
@@ -0,0 +1,70 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from time import sleep
+
+from scrubber_test_base import TestWithScrubber
+
+
+class TestWithScrubberFreq(TestWithScrubber):
+    # pylint: disable=too-many-nested-blocks
+    """Test the scrubber frequency option.
+
+    :avocado: recursive
+    """
+    def test_objects_scrubbed_properly(self):
+        """JIRA ID: DAOS-7371
+            Test scrubber frequency can be dynamically updated.
+
+            1. Create a test pool without scrubber properties.
+               The default scrub frequency is set to 1 week.
+            2. Initial scrub happens when the objects are created.
+            3. Check objects are not scrubbed after initial scan.
+               Wait for 5 minutes and see scrubber not doing
+               any more object scrubbing now.
+            4. Update the scrubber frequency to 5 seconds.
+            5. Observe the objects are scrubbed every 5 seconds.
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=scrubber
+        :avocado: tags=TestWithScrubberFreq,test_objects_scrubbed_properly
+        """
+        pool_prop = self.params.get("properties", '/run/pool/*')
+        cont_prop = self.params.get("properties", '/run/container/*')
+        self.create_pool_cont_with_scrubber(pool_prop=pool_prop, cont_prop=cont_prop)
+        # Run IOR and gather the total scrubbed metrics information.
+        self.run_ior_and_check_scruber_status(pool=self.pool, cont=self.container)
```

Comment:
FYI this pre-existing function has a typo
```suggestion
        self.run_ior_and_check_scrubber_status(pool=self.pool, cont=self.container)
```

Reply:
Great... You spotted it... Sigh... Will push the changes.

Reply:
lol

Reply:
I think it was the mdtest one that you fixed before.

Reply:
The problem was I changed the implementation on the final mdtest script (Which was checked in) without interfering with the scrubber_test_base.py file (where the typo error was there).